src/aioquic/quic/configuration.py

```diff
@@ -46,6 +46,11 @@ class QuicConfiguration:
     Connection-wide flow control limit.
     """
 
+    max_packet_size: int = 1280
+    """
+    Maximum size of quic packets.
```

Comment:
QUIC in capitals, and it feels as though we should be more explicit about what we mean by "packet size". (without UDP overhead, etc..)